### PR TITLE
fix: Issues with copying published charts

### LINKED ISSUE #1321: Browser Tab Closes when chart is published
See: https://int.visualize.admin.ch/v/EbXixKlDKslp?dataSource=Int 
To Reproduce: 
1. COpy edit VIsualization 
2. Publish visulisation 
3. see browser tab closes

### PR DESCRIPTION
**Pull request by bprusinowski**
Fixes #1321.

### PR deployment (ok)
1. Go to [this link](https://visualization-tool-git-fix-wrong-config-props-setting-ixt1.vercel.app/en/create/new?cube=https://environment.ld.admin.ch/foen/ubd000502/4&dataSource=Prod).
2. Change chart type to scatterplot.
3. ✅ See that the segment is initialized automatically with `Hierarchy` dimension.
4. Change segment dimension to `Year`.
5. Publish.
6. Click on `Copy and edit this visualization`.
7. ✅ See that the chart correctly loaded into the editor mode.
8. Switch to a column chart.
9. ✅ See that the segment is empty.

### TEST (not ok)
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://environment.ld.admin.ch/foen/ubd000502/4&dataSource=Prod).
2. Change chart type to scatterplot.
3. ❌ See that no segment is selected by default.
4. Select `Hierarchy` dimension as a new segment.
5. Switch to `Year` dimension as a new segment.
6. Publish.
7. Click on `Copy and edit this visualization`.
8. ❌ See that it's not possible to copy the chart (due to wrong chart config we are redirected to a browse page).
---
1. Go to [this link](https://test.visualize.admin.ch/en/create/new?cube=https://environment.ld.admin.ch/foen/ubd000502/4&dataSource=Prod).
2. Change chart type to scatterplot.
3. Switch to `Year` dimension as a new segment.
4. Switch to a column chart.
5. ❌ See that `Year` is both used as X axis and segmentation field.

### CODE REVIEW

File: e2e/filters.spec.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ describe("Filters", () => {
 
     await filters.locator("label").first().waitFor({ timeout: 30_000 });
 
+    // Give some time for the filters to load
+    await sleep(4_000);
```
Comment (ptbrowne) [inline e2e/filters.spec.ts:22]: Can we wait for all the filters using the same kind of technique as above (waiting for the labels ?) ?

**Review by ptbrowne — DISMISSED** [review]

File: e2e/filters.spec.ts (hunk 1 above)
Comment (bprusinowski) [reply e2e/filters.spec.ts:22]: We would need to adjust the logic, as the filter elements are visible, but they can have empty value during loading, so we'd need to wait until every input element has non empty value 👍

I can to try do it, any tips are welcome if you already have the approach in mind, not sure yet how to solve this 🙏

Comment (bprusinowski) [reply e2e/filters.spec.ts]: I think I managed to do it in https://github.com/visualize-admin/visualization-tool/pull/1387/commits/d73e35602def168e38f095d5d413c47649c7e6ac

**Review by ptbrowne — APPROVED** [review]
Thanks 👍

**Pull request merged**